app/components/content-container.tsx

```diff
@@ -20,6 +20,7 @@ export function ContentContainer({
     "items-center",
     "justify-center",
     "space-y-3",
```

Comment:
I hated how this element stretched across the entire width of the screen at huge screen sizes. Chopping it down a bit on the high end.